src/extension.ts

```diff
@@ -527,6 +531,42 @@ function clearLocations() {
   issueLocationsView.message = null;
 }
 
+interface IndexQP extends VSCode.QuickPickItem {
+  index: number;
+}
+
+async function ShowCDOptions(paths: VSCode.Uri[]) {
```

Comment:
```suggestion
async function showCompilationDatabaseOptions(paths: VSCode.Uri[]) {
```
Function name should start with a lower case `s`, and I believe that `CD` is a bit obscure to someone reading the code.